utils/EventRepository.js

```diff
@@ -227,12 +230,30 @@ class EventRepository {
 
     // modify the url for event specific endpoints
     const url = this.url.slice(-1) == "/" ? this.url.slice(0, -1) : this.url;
-    // add items to the queue
-    this.payloadQueue.addItem({
-      url: `${url}/v1/${type}`,
-      headers,
-      message,
-    });
+    if (this.useBeacon) {
+      const targetUrl = `${url}/beacon/v1/batch`;
```

Comment:
Can you verify if we have to send it to the Beacon batch endpoint or the regular batch endpoint works?